pyspecdata/core.py

```diff
@@ -3391,6 +3402,8 @@ def mean(self, *args, **kwargs):
                 thiserror = np.std(self.data, axis=thisindex)
                 if np.isscalar(thiserror):
                     thiserror = r_[thiserror]
+                if return_stderr:
+                    thiserror /= np.sqrt(self.data.shape[thisindex])
```

Comment:
see above

Reply:
I don't think this is explained properly above, but this one is easiest to just explain here --> if `return_stderr` is set, then it will return the standard error rather than the standard deviation.

In making this comment, I realize that we should probably make `ddof` (see [here](https://numpy.org/doc/stable/reference/generated/numpy.std.html) for explanation) a kwarg of the main function, pass it to std, and default it to 1 when we ask for stderr and 0 when we as for stdev.

Reply:
should we do that in this PR or you're just commenting for reference?